assets/test/cmake/compile_commands.json

```diff
@@ -0,0 +1,7 @@
+[
+{
+  "directory": "/Users/adamfowler/Developer/tools/vscode-swift/assets/test/cmake",
```

Comment:
Do we want these committed?

Reply:
We need the compile_commands.json to trigger the extension for CMake projects. I'll edit it though

Reply:
I've removed most of the cmake project files as they were not needed